_posts/0000-01-02-catgonza.md

```diff
@@ -0,0 +1,2 @@
+layout: slide
+title: "Welcome to our second slide!"
```

Comment:
Hello! Thanks for joining my pull request. Please only make useful changes that will make me be the best web developer on earth ^^